mod_reforged/hooks/config/tip_of_the_day.nut

```diff
@@ -19,6 +19,7 @@ for (local index = (::Const.TipOfTheDay.len() - 1); index >= 0; index--)
 }
 
 ::Const.TipOfTheDay.extend([
+	"Dismissing a brother will transfer all of their equipped items into your stash if you have space."
```

Comment:
This is unnecessary imo as this information is present inside the Dismiss dialogue box. So the first time a new player sees that box after clicking Dismiss they will learn it.

<img width="410" alt="image" src="https://github.com/Battle-Modders/mod-reforged/assets/55047920/b04f3d0b-8c7f-458c-af69-fa91577aa084">